setup.py

```diff
@@ -21,7 +20,8 @@
     "gunicorn<20.1; python_version < '3.6'",
     "gunicorn==20.1.*; python_version >= '3.6'",
 }
-uwsgi_extras = {"uwsgi==2.0.*"}
+uwsgi_max_extras = {"uwsgi<2.1"}
```

Comment:
mmmm any reason to use <2.1 instead of original 2.0.*? I don't want us to get confused with the min version

Reply:
so honestly, I was worried that `2.0.*` wouldn't match `2.0.19.1` but it looks like it does. I'll revert this